astro/src/content/docs/apis/passwordless.mdx

```diff
@@ -111,7 +121,7 @@ Available Since Version 1.11.0.
 
 <Aside type="caution" title="Deprecated usage">
   * This usage of the API may be removed at a later date. Once available prefer the usage outlined above which is available beginning in `1.11.0` which means you will need to use the Start API prior to calling this API.
-  * Passwordless login with phone numbers, introduced in version `1.99.9`, is not supported by this API. Use the [Start Passwordless Login](#start-passwordless-login) API to obtain a code and then provide that code to this API (see above).
+  * Passwordless login with phone numbers, introduced in version `1.99.9`, is not supported by this usage of this API endpoint. Use the [Start Passwordless Login](#start-passwordless-login) API to obtain a code and then provide that code to this API (see above).
```

Comment:
Maybe unnecessary but I didn't want to scare people off the whole API, just this usage of the API.